core/types/transaction_signing.go

```diff
@@ -550,7 +550,7 @@ func deriveChainId(v *big.Int) *big.Int {
 	if v.BitLen() <= 64 {
 		v := v.Uint64()
 		if v == 27 || v == 28 {
-			return new(big.Int)
+			return big.NewInt(1337) // TODO: verify proper handling
```

Comment:
What's going on here?

Reply:
Very careful touching the transaction_signing since all the transactions and externalBlocks validation depends on it. I don't see a need to touch it at all.

Reply:
This change has been reverted. Resolving.